bpf/bpf_lxc.c

```diff
@@ -544,13 +544,23 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 
 	tuple.nexthdr = ip4->protocol;
 
-	if (unlikely(!is_valid_lxc_src_ipv4(ip4)))
+	if (unlikely(!is_valid_lxc_src_ipv4(ip4))) {
+#ifndef ENABLE_DHCP_MESSAGES
```

Comment:
I think there are use cases for this besides DHCP, so maybe we should make it a bit more generic and allow all srcIP spoofing? I'm also worried of the impact `is_valid_dhcpv4_message` could have on the BPF complexity.

Reply:
you mean you are thinking about having this as a part of `is_valid_lxc_src_ipv4` func - for all packets? Also does it make sense to you to switch `#ifndef` for `#ifdef` for better extensibility/readability?

Reply:
I mean we should just accept packets if `ENABLE_DHCP_MESSAGES` is defined, regardless of whether or not they are DHCP packets.